easybuild/easyconfigs/p/partitionfinder/partitionfinder-2.1.1-intel-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,49 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Tarball'
+
+name = 'partitionfinder'
+version = '2.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/brettc/partitionfinder/releases/tag/v2.1.1'
```

Comment:
https://www.robertlanfear.com/partitionfinder is better to use as homepage imho (since it also points to a tutorial, etc.)